charts/access/jira/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: teleport-plugin-jira
+description: A Helm chart for the Teleport Jira Plugin
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: "9.1.3"
```

Comment:
We're up to 9.3.4 as of https://github.com/gravitational/teleport-plugins/commit/8894eb7556983be7cc1432b20939589131e7fe5d